src/applications/personalization/profile-2/components/ProfileWrapper.jsx

```diff
@@ -60,23 +61,52 @@ class ProfileWrapper extends Component {
     </div>
   );
 
+  createBreadCrumbAttributes = () => {
+    const { location, route } = this.props;
+    const activeLocation = location?.pathname.replace('/', '');
+    const childRoutes = route?.childRoutes;
+    const activeRoute = childRoutes.find(
+      childRoute => childRoute.path === activeLocation,
+    );
+
+    const activeRouteName = activeRoute?.name;
+
+    return { activeLocation, activeRouteName };
+  };
```

Comment:
I've never been great at the whole React routing subject (seems like the API changes heavily every version!) - would it be an option at all to use a Redux `Link` for any of this, and possibly get some of the active-route handling for free?

Reply:
Hmm, interesting but I am not sure I understand - do you mean Link from `react-router`? What do you mean by `possibly get some of the active-route handling for free`? Do you have an example of where we get that handling elsewhere in the app?

Reply:
Something like this - https://github.com/department-of-veterans-affairs/vets-website/pull/12376/files#r416917740

Reply:
`src/applications/vaos/components/Breadcrumbs.jsx` looks like a potential example

Reply:
It looks like the `<Link />` prop `activeClassName` would be useful but only in situations where we're conditionally applying an active class name. For Breadcrumbs we don't do that, so I don't think it would be useful here?

Reply:
Good point, I think I was getting mixed up with `aria-current="page"` so that the active link just shows as text and isn't clickable (to prevent just reloading the same page)

Reply:
tangent: I was wondering how to make the links in the new Profile sidenav non-clickable when its associated page was active. It's not as easy as you'd like, but it also doesn't seem to matter in the context of a React Router app (or anything using client-side routing) since "changing" the client route from '/profile/route/` to `/profile/route/` is effectively a no-op; nothing visual happens and nothing on the page is reloaded.